CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
-# "More Modern" CMake version
+# We require this to get object library link library support and
+# combined python 2 + 3 support
 
```

Comment:
copy'n'paste bug?

Reply:
yeah, i had noticed that and pushed a fix over top.